From cdfb9ccdfb1fb53f9fd71227b91fa6b197edfc44 Mon Sep 17 00:00:00 2001
From: Robert Mader <robert.mader@collabora.com>
Date: Tue, 30 Sep 2025 23:13:52 +0200
Subject: [PATCH 2/3] aperture: Rename video-specific variables

In preparation of the next commit.

(cherry picked from commit 9787e31b8eb7b89baae7cc1dc48aab5d51b4e162)
---
 aperture/src/viewfinder.rs | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/aperture/src/viewfinder.rs b/aperture/src/viewfinder.rs
index 9d0c981..bf998e8 100644
--- a/aperture/src/viewfinder.rs
+++ b/aperture/src/viewfinder.rs
@@ -898,7 +898,7 @@ impl Viewfinder {
         use gst_pbutils::{ElementProperties, ElementPropertiesMapItem};
 
         // Video encoder properties
-        let element_properties_map = ElementProperties::builder_map()
+        let video_properties_map = ElementProperties::builder_map()
             .item(
                 ElementPropertiesMapItem::builder("x264enc")
                     .field("bitrate", DEFAULT_BITRATE)
@@ -948,7 +948,7 @@ impl Viewfinder {
             )
             .build();
 
-        let profile = match self.video_format() {
+        let video_profile = match self.video_format() {
             VideoFormat::H264Mp4 => {
                 let mut hw_encoder_found = false;
                 let registry = gst::Registry::get();
@@ -994,7 +994,7 @@ impl Viewfinder {
                     &gst::Caps::builder("video/x-h264").build(),
                 )
                 .variable_framerate(true)
-                .element_properties(element_properties_map)
+                .element_properties(video_properties_map)
                 .build();
                 container_profile = container_profile.add_profile(video_profile);
 
@@ -1054,7 +1054,7 @@ impl Viewfinder {
                 )
                 .preset("Profile Realtime")
                 .variable_framerate(true)
-                .element_properties(element_properties_map)
+                .element_properties(video_properties_map)
                 .build();
                 container_profile = container_profile.add_profile(video_profile);
 
@@ -1071,7 +1071,7 @@ impl Viewfinder {
         };
 
         let camerabin = self.imp().camerabin();
-        camerabin.set_property("video-profile", profile);
+        camerabin.set_property("video-profile", video_profile);
     }
 
     fn init(&self) {
-- 
2.51.0

